packages/docusaurus-1.x/lib/server/liveReloadServer.js

```diff
@@ -27,7 +27,21 @@ function start(port) {
     },
   );
 }
+
 const getReloadScriptUrl = () => {
+  // Please, JS programmer show me how to save state of 'program' globally for each
```

Comment:
Use `process.env.LIVERELOAD_HOST` instead of hacking into `commander`?

Reply:
I have decided to go another way for live reload, so I am withdrawing this PR.